src/core/core.platform.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// Core.Platform abstracts away browser specific APIs from the chart
+module.exports = function(Chart) {
+	var platform = Chart.corePlatform = {};
```

Comment:
Yeah, that was actually my first thought before I made it more OO like. I'll move to a static class in `Chart.Platform`